interfaces/builtin/multipass_support.go

```diff
@@ -0,0 +1,156 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2019 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const multipassSupportSummary = `allows operating as the Multipass service`
+
+const multipassSupportBaseDeclarationPlugs = `
+  multipass-support:
+    allow-installation: false
+    deny-auto-connection: true
+`
+
+const multipassSupportBaseDeclarationSlots = `
+  multipass-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const multipassSupportConnectedPlugAppArmor = `
+# Description: allow operating as the Multipass daemon.
+
+# Multipass is a tool to create and manage Virtual Machines and their images.
+# Each VM runs as a separate "qemu" process (on Linux). VM images are automatically
+# downloaded, but need conversion using "qemu-img". Networking is provided by
+# configuring a TUN/TAP network on the host, with DHCP provided by a shared
+# "dnsmasq" process. File sharing between the VM and the host is provided by a
+# "sshfs_server" utility. All these utilities are shipped in the snap.
+
+# Each of these utilities have a very different different purpose, and an attempt
+# to confine them all in a single profile would result in an extremely broad AppArmor
+# profile.
+
+# Instead we defer to Multipass the responsibility of generating custom AppArmor
+# profiles for each of these utilities, and trust it launches each utility with
+# all possible security mechanisms enabled. The Multipass daemon itself will run
+# under this restricted policy.
+
+# Therefore this policy intentionally allows the Multipass daemon to configure
+# AppArmor.
```

Comment:
This comment is great but a little much for a policy comment. I suggest moving it (everything from under Description to here) to a code comment and then expand on the inline comment slightly.